lib/includes/template-functions.php

```diff
@@ -1861,27 +1882,40 @@ function epl_switch_views() {
  *
  * @since 2.0
  * @since 3.3 Revised.
+ * @since 3.5 gets the shortcode attributes as param 
  */
-function epl_sorting_tool() {
+function epl_sorting_tool( $attributes = [] ) {
+
+        $instance_id = !empty( $attributes['instance_id'] ) ? sanitize_text_field( $attributes['instance_id'] ) : '';
 	$sortby = '';
+        $set_instance_id = '';
 	if ( ! empty( $_GET['sortby'] ) ) {
 		$sortby = sanitize_text_field( wp_unslash( $_GET['sortby'] ) );
 	}
+        if ( ! empty( $_GET['instance_id'] ) ) {
+		$set_instance_id = sanitize_text_field( wp_unslash( $_GET['instance_id'] ) );
+	}
 	$sorters = epl_sorting_options();
 	?>
 
 	<div class="epl-loop-tool epl-tool-sorting epl-properties-sorting epl-clearfix">
-		<select id="epl-sort-listings">
+		<select class="epl-sort-listings" id="epl-sort-listings" data-instance-id="<?php echo esc_attr( $instance_id ); ?>">
```

Comment:
Caution using id= without passing somehing unique. EG we can have these on the same page multiple times

Pass the $instance_id

e.g

id="epl-sort-listings-<?php echo esc_attr( $instance_id ); ?>"